php/content.md

```diff
@@ -6,7 +6,7 @@ PHP is a server-side scripting language designed for web development, but which
 
 %%LOGO%%
 
-# How to use this image.
```

Comment:
Aligns headline with other README (which don't use a dot at the end).

See https://github.com/docker-library/docs/tree/master/wordpress#how-to-use-this-image.